### PR TITLE
Update to 2.11.0

### PR DESCRIPTION
**Pull request by hfiguiere**
- add clap plugin

### CODE REVIEW

File: org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "id": "org.freedesktop.LinuxAudio.Plugins.ChowTapeModel",
     "branch": "21.08",
     "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
-    "runtime-version": "21.08",
+    "runtime-version": "stable",
```
Comment (Bleuzen) [inline org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json:5]: what exactly this means?
so for example plugin built against 21.08 sdk can be used in future 22.08 runtime when it becomes stable?

**Comment by Bleuzen** [comment]
bot, build

File: org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json (hunk 1 above)
Comment (hfiguiere) [reply org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json:5]: this is the base app. It's a recent change and should be that way.

Comment (Bleuzen) [reply org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json:5]: > this is the base app. It's a recent change and should be that way.

great news :)

**Pull request merged**